flopy/utils/geospatial_utils.py

```diff
@@ -453,7 +453,7 @@ def shapely(self):
         return self._shapely
 
     @property
-    def geo_dataframe(self):
+    def geodataframe(self):
```

Comment:
why `to_geodataframe...` elsewhere and not here, other than the fact the old property didn't have `to_...`?

suggest also deprecating the old one to avoid breakage?